examples/vertx-opentracing/README.md

```diff
@@ -135,7 +135,7 @@ See [Example Requests](#example-requests) for further commands.
 
 ## Example requests
 
-For the following examples, we'll use `${ORDER_MANAGER_SERVER}` to refer to the Hawkular URI. It might be a simple hostname,
+For the following examples, we'll use `${ORDER_MANAGER_SERVER}` to refer to the example's server. It might be a simple hostname,
 like when it's deployed on OpenShift, or `hostname:port`, like when it's deployed locally (`localhost:8080`, for instance).
```

Comment:
Only one change - if you could make it `8180` as this is the port outside openshift.